agents/craftassist/tests/fake_agent.py

```diff
@@ -479,14 +491,21 @@ def safe_get_changed_blocks(self) -> List[Block]:
     ## World setup
     ######################################
 
-    def set_blocks(self, xyzbms: List[Block], origin: XYZ = (0, 0, 0)):
+    def set_blocks(self, xyzbms: List[Block], boring_blocks: Tuple[int], origin: XYZ = (0, 0, 0)):
```

Comment:
as above: could we have a memory.update(dict_with_changes}), and not call e.g. memory.maybe_add_block_to_memory here (but in the memory's .update(data)

Reply:
I think we could still run all of these in a memory.update(changes) where changes is a dict with the appropriate data

Reply:
the memory.update(changes) could ofc choose to not do anything based on the exact changes passed in